data/saas/config/gong_config.yml

```diff
@@ -0,0 +1,46 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Gong
+  type: gong
+  description: A sample schema representing the Gong connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+    default: api.gong.io
```

Comment:
Typo here, it should be `default_value` and make sure to tab it in to line up with `name`